docs/sources/operators-guide/deploying-grafana-mimir/jsonnet/memberlist.md

```diff
@@ -0,0 +1,147 @@
+---
+title: "Migrating from Consul to Memberlist KV store for hash rings"
+menuTitle: "Migrating from Consul to Memberlist"
+description: "Learn how to migrate from using Consul as KV store for hash rings to using memberlist without any downtime."
+weight: 30
```

Comment:
So we keep it at the end:

```suggestion
weight: 40
```

Reply:
I take your word for it, 30 seemed unused before, but if 40 is better, let's use 40 :)